common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -212,9 +212,11 @@ def _gitlab_repo_contents(repo, path, commit)
         gitlab_client.
           repo_tree(repo, path: path, ref_name: commit, per_page: 100).
           map do |file|
+            #GitLab API essentially returns the output from `git ls-tree`
```

Comment:
oh just a tiny style nit:
```suggestion
            # GitLab API essentially returns the output from `git ls-tree`
```